client/blocks/edit-gravatar/index.jsx

```diff
@@ -4,6 +4,7 @@
 import React, { Component, PropTypes } from 'react';
 import classnames from 'classnames';
 import { connect } from 'react-redux';
+import debug from 'debug';
```

Comment:
Ok, will use that instead :) I saw the debug/log idea floating around, too. We should probably update our docs and/or linters to help with consistency. _/me writes it down in my todos_